forte/data/ontology/top.py

```diff
@@ -1168,12 +1172,125 @@ def __init__(
         )
 
 
+class Payload(Entry):
+    """
+    A payload class that holds data cache of one modality and its data source uri.
+
+    Args:
+        pack: The container that this `Payload` will
+            be added to.
+        modality: modality of the payload such as text, audio and image.
+        payload_idx: the index of the payload.
```

Comment:
Should provide informative docstring. For example, `the index of the payload` does not add any value to `payload_idx`